### ISSUE TITLE
Missed communication with Application catalog (Openstack murano)

### ISSUE THREAD

**Issue by nmakhotkin** [ContainX/openstack4j#830]
Our team is planning to work with Openstack on Java but we need communication with murano. 

We start the development of this part of openstack4j soon.

**Comment by auhlig** [comment]
Hey @nmakhotkin,
Very cool. Looking forward to the PR.
Could you also include tests? Feel free to reach out in case you need help.

**Comment by nmakhotkin** [comment]
Hi @auhlig,

yes, I included unit-test for that functionality. Did you mean some integration-tests?

**Comment by auhlig** [comment]
As you started with the unit tests that's fine. 

Could you also update the website documentation https://github.com/ContainX/openstack4j.com ?

**Comment by nmakhotkin** [comment]
Yes, I'll update it once I make the murano support more or less complete (I plan to do it iteratively)

**Comment by auhlig** [comment]
Hi @nmakhotkin,
Many thanks for all the contributions.

I recently updated the integration tests (incl. readme). Since you mentioned that you're also testing murano service implementation in os4j against an openstack instance I'd like to ask if you'd mind adding some integration tests. That would also give me some feedback on how usable it is. 
What do you think?

**Comment by nmakhotkin** [comment]
@auhlig, sorry but I can't do it now.  Our company suddenly changed the vector of development and unfortunately, murano-apps (where we needed murano-jenkins-plugin) development is also closed now. Sorry.

I can only add the rest of functionality and integration tests in free time, try to do that.

**Comment by auhlig** [comment]
Oh. What a pity! :(
Again thanks for all the work you did so far.

Cool. Feel free to reach out if we can help.

**Comment by vinodborole** [comment]
@nmakhotkin thnx buddy; looking forward for your response on my email i sent the other day.
